core/drand_beacon.go

```diff
@@ -120,6 +121,11 @@ func (bp *BeaconProcess) Load() (bool, error) {
 	bp.log = bp.log.Named(fmt.Sprint(bp.index))
 
 	bp.log.Debugw("", "serving", bp.priv.Public.Address())
+	if bp.group == nil {
+		metrics.DKGStateChange(metrics.DKGNotStarted, metrics.UnknownBeaconID, false)
+	} else {
+		metrics.DKGStateChange(metrics.DKGNotStarted, bp.group.ID, false)
```

Comment:
This is a bit different since it's not that we don't have a beacon ID, it's that we don't even have a group. Not having dived deep into the code I don't know why that might be, I found it because I got a crash during the tests. So you're saying that `bp.group` being `nil` is the same as `bp.group.id` being `""`?

Reply:
No, I'm saying it can be non-`nil`, but still be the empty string `""`.
So it might be "nicer" to have a check and replace it with the `common.DefaultBeaconID` when it's the empty string, so that we never have an empty beaconID in our logs / graphs and instead see the `common.DefaultBeaconID` name in our dashboard.

Reply:
oh I now realize what you meant. Fixing it.